src/api/custom-resolvers/mutations/create-an-account/index.test.ts

```diff
@@ -42,95 +47,322 @@ describe('custom-resolvers/create-an-account', () => {
     jest.resetAllMocks();
   });
 
-  beforeEach(async () => {
-    jest.resetAllMocks();
+  describe('when an account with the provided email does NOT already exist', () => {
```

Comment:
Note: there's a lot of repeated `beforeAll` spys/setup in this file, would like to clean up after this PR. Will create a ticket.